impeller/entity/contents/content_context.h

```diff
@@ -135,23 +130,27 @@ using TextureStrictSrcPipeline =
 using TiledTexturePipeline =
     RenderPipelineHandle<TextureUvFillVertexShader,
                          TiledTextureFillFragmentShader>;
```

Comment:
Changes here is the gaussian, morph, and border mask filters all have identical vertex stages so they can share them now.

Likewise, vertices_uber and porter duff have identical vertex stages.

Also we don't need two gaussians, only ever need one.